packages/fury-adapter-swagger/test/generator-test.js

```diff
@@ -53,7 +53,7 @@ describe('bodyFromSchema', () => {
     const asset = bodyFromSchema(schema, payload, parser, 'application/json');
 
     expect(typeof asset.content).to.equal('string');
-    expect(asset.content.length).to.be.above(0);
+    expect(asset.content.length).to.equal(0);
```

Comment:
This is now constantly an empty string because the change to randomness, this now matches the behaviour of API Blueprint where an element of type string without any example, default values would emit an empty string.